src/sentry/static/sentry/app/views/organizationDiscover/styles.jsx

```diff
@@ -139,7 +179,7 @@ export const Heading = styled.h2`
 `;
 
 export const Fieldset = styled.fieldset`
-  margin: ${space(3)};
+  margin: ${space(3)} ${space(3)} 0;
```

Comment:
Does this affect other places where Fieldset is used?

Reply:
It's used in savedQueryList, but it looks fine to me!